authors/apps/profiles/tests/test_profile.py

```diff
@@ -1,8 +1,7 @@
-
+from django.urls import reverse
```

Comment:
Insert one blank line between Django imports and rest_framework imports.

Reply:
This comment is of no use now since you're separating your imports with one blank line.